src/Client.php

```diff
@@ -75,7 +75,8 @@ public function post($url, $organizationId, array $data = [], array $params = []
         return $this->processResult($this->httpClient->post(
             $url,
             [
-                'query' => $this->getParams($organizationId, $data) + $params,
+                'query' => $params + $this->getParams($organizationId),
+                'form_params' => ['JSONString' => json_encode($data)],
```

Comment:
Sorry @shandyDev, I was misunderstanding the `depth` argument for `json_encode()`. This is not intended to take into account the object length but the nested recursion in the structure, so I think this can be safely used in all of the calls.